tests/e2e/deploy-kubestone.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -x
+
+# Copyright 2019 The xridge kubestone contributors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+# FIXME: Should use an other tag to avoid confusion
```

Comment:
Please use TODO tag so that the bot can pick it up.

Reply:
done